service/controller/resource/deployment/resource.go

```diff
@@ -131,6 +130,18 @@ func isDeploymentModified(a, b *v1.Deployment) bool {
 	return false
 }
 
+func isDeploymentModified(a, b *v1.Deployment) bool {
+	if isAnnotationModified(a, b, key.VersionBundleVersionAnnotation) {
+		return true
+	}
+
+	if isAnnotationModified(a, b, key.ReleaseVersionAnnotation) {
+		return true
+	}
+
+	return false
```

Comment:
```suggestion
	return isAnnotationModified(a, b, key.VersionBundleVersionAnnotation) || 
		isAnnotationModified(a, b, key.ReleaseVersionAnnotation)
```

Reply:
To be honest I like the current version more. The line gets a bit too long and confusing if we merge the checks.

Reply:
Works for me.